src/commands/analyze/command.ts

```diff
@@ -256,22 +258,36 @@ export async function analyzeCommand(options: GenezioAnalyzeOptions) {
             );
             debugLogger.debug("Serverless HTTP entry file found:", entryFile);
 
-            // TODO: Add support for detecting and building typescript backends
-            // const isTypescriptFlag = await isTypescript(contents);
+            const isTypescriptFlag = await isTypescript(contents);
+            debugLogger.debug("Is Serverless HTTP typescript:", isTypescriptFlag);
+
+            let entryFileBuildOut = entryFile;
+            if (isTypescriptFlag) {
+                const tsconfigJsonContent = JSON.parse(
+                    await retrieveFileContent(path.join(componentPath, "tsconfig.json")),
+                );
```

Comment:
nit - I think you can do this inside `getEntryFileTypescript` for a cleaner code.